src/network/networkd-link.c

```diff
@@ -318,6 +318,19 @@ static int link_enable_ipv6(Link *link) {
         return 0;
 }
 
+static bool link_is_enslaved(Link *link) {
+        if (!link->enslaved)
+                return false;
+
+        if (link->network && link->network->bond && link->flags & IFF_SLAVE)
```

Comment:
hmm, I find the mixture of && and & hard to read. maybe use FLAGS_SET() for the IFF_SLAVE check?

Reply:
Indeed. The line seems confusing :-p